### PR TITLE
add article about how to Manage dotfiles with Git

### PR DESCRIPTION
**Pull request by sogalgeeko**
This is a simple article about how to manage your dotfiles simply with Git.
It has been heavily inspired by articles found around on the Web and cited in sources.

### CODE REVIEW

**Comment by hellcp** [comment]
If you could place the image in a subdirectory called `2019-03-27` for convenience please :P 

You can also see the preview https://deploy-preview-23--news-o-o-preview.netlify.com/2020/03/27/manage-dotfiles-with-git/

**Comment by sogalgeeko** [comment]
> If you could place the image in a subdirectory called `2020-03-27` for convenience please :P

Done. Will update the README.md to add this.

> You can also see the preview https://deploy-preview-23--news-o-o-preview.netlify.com/2020/03/27/manage-dotfiles-with-git/

Thanks.

**Comment by hellcp** [comment]
From my pov, this is done, I will let the marketing team merge this

File: _posts/2020-03-27-Manage-dotfiles-with-Git.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Tumbleweed
+- git 
+- configuration
+- shell
+
+---
+
+## Dot what ???
+
+What is commonly referred to as *dotfiles* are all those small plain text files that contains your softwares' configuration. Most of the time they reside in your `$HOME` directory but are hidden as they are prefixed with a dot, hence their name. For some apps, you can find them as well in the `$HOME/.config` directory.
```
Comment (vinzv) [inline _posts/2020-03-27-Manage-dotfiles-with-Git.md]: Use plural instead of singular: […] that contain<b>s</b> your […]

Comment (sogalgeeko) [reply _posts/2020-03-27-Manage-dotfiles-with-Git.md]: Fixed.
I also updated the README.md after LCP's remark: I add a line about where to put images used in submitted articles.

**Pull request merged**